apps/src/StudioApp.js

```diff
@@ -446,6 +445,7 @@ StudioApp.prototype.handleSharing_ = function (options) {
     }
     if (belowVisualization) {
       if (options.noButtonsBelowOnMobileShare) {
+        var visualization = document.getElementById('visualization');
         belowVisualization.style.display = 'none';
         visualization.style.marginBottom = '0px';
```

Comment:
It was working. apparently when we do container.innerHTML = foo, any HTML in foo that has an id gets loaded into the global namespace. This seems crazy to me, but it's why both this and runButton (the next change) worked before.